lib/rucio/core/transfer.py

```diff
@@ -1156,6 +1222,7 @@ def build_transfer_paths(
         PreferDiskOverTape.external_name,
         PathDistance.external_name,
         PreferSingleHop.external_name,
+        FailureRate.external_name,
```

Comment:
Don't add this strategy to the default ones. We must not change the default behavior. Users would be surprised to see the behavior of the program change so radically after an upgrade.